### PR TITLE
NetAdapter reference update for WS2022

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: docset/winserver2022-ps/netadapter/Disable-NetAdapterUso.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to
+run the cmdlet. If this parameter is omitted or a value of `0` is entered, then
+Windows PowerShell&reg; calculates an optimum throttle limit for the cmdlet based on
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Disable-NetAdapterUso.md:226]:
```suggestion
Windows PowerShell calculates an optimum throttle limit for the cmdlet based on
```

File: docset/winserver2022-ps/netadapter/Enable-NetAdapterUso.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+## PARAMETERS
+
+### -AsJob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+ The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+ For more information about Windows PowerShell&reg; background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Enable-NetAdapterUso.md:61]:
```suggestion
 For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2022-ps/netadapter/Enable-NetAdapterUso.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to
+run the cmdlet. If this parameter is omitted or a value of `0` is entered, then
+Windows PowerShell&reg; calculates an optimum throttle limit for the cmdlet based on
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Enable-NetAdapterUso.md:225]:
```suggestion
Windows PowerShell calculates an optimum throttle limit for the cmdlet based on
```

File: docset/winserver2022-ps/netadapter/Get-NetAdapterDataPathConfiguration.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+## PARAMETERS
+
+### -AsJob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+ The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+ For more information about Windows PowerShell&reg; background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Get-NetAdapterDataPathConfiguration.md:55]:
```suggestion
 For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2022-ps/netadapter/Get-NetAdapterDataPathConfiguration.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to
+run the cmdlet. If this parameter is omitted or a value of `0` is entered, then
+Windows PowerShell&reg; calculates an optimum throttle limit for the cmdlet based on
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Get-NetAdapterDataPathConfiguration.md:178]:
```suggestion
Windows PowerShell calculates an optimum throttle limit for the cmdlet based on
```

File: docset/winserver2022-ps/netadapter/Get-NetAdapterUso.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+## PARAMETERS
+
+### -AsJob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+ The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+ For more information about Windows PowerShell&reg; background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Get-NetAdapterUso.md:66]:
```suggestion
 For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2022-ps/netadapter/Get-NetAdapterUso.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to
+run the cmdlet. If this parameter is omitted or a value of `0` is entered, then
+Windows PowerShell&reg; calculates an optimum throttle limit for the cmdlet based on
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Get-NetAdapterUso.md:152]:
```suggestion
Windows PowerShell calculates an optimum throttle limit for the cmdlet based on
```

File: docset/winserver2022-ps/netadapter/Set-NetAdapterDataPathConfiguration.md [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+## PARAMETERS
+
+### -AsJob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+ The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+ For more information about Windows PowerShell&reg; background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Set-NetAdapterDataPathConfiguration.md:64]:
```suggestion
 For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2022-ps/netadapter/Set-NetAdapterDataPathConfiguration.md [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to
+run the cmdlet. If this parameter is omitted or a value of `0` is entered, then
+Windows PowerShell&reg; calculates an optimum throttle limit for the cmdlet based on
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Set-NetAdapterDataPathConfiguration.md:205]:
```suggestion
Windows PowerShell calculates an optimum throttle limit for the cmdlet based on
```

File: docset/winserver2022-ps/netadapter/Set-NetAdapterUso.md [hunk 10]
```diff
@@ -0,0 +1,284 @@
...
+## PARAMETERS
+
+### -AsJob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+ The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+ For more information about Windows PowerShell&reg; background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Set-NetAdapterUso.md:60]:
```suggestion
 For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2022-ps/netadapter/Set-NetAdapterUso.md [hunk 11]
```diff
@@ -0,0 +1,284 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to
+run the cmdlet. If this parameter is omitted or a value of `0` is entered, then
+Windows PowerShell&reg; calculates an optimum throttle limit for the cmdlet based on
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netadapter/Set-NetAdapterUso.md:224]:
```suggestion
Windows PowerShell calculates an optimum throttle limit for the cmdlet based on
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PatAltimore, kindly commit the suggested changes to remove the registered trademark symbols. Many thanks.

**Comment by PatAltimore** [comment]
Hi @JohanFreelancer9,

Thanks for catching the issue. I've committed your suggestions.

Thanks,
Pat

**Comment by PatAltimore** [comment]
Hi @get-itips,

Could you review this PR when you get a chance?

Thanks,
Pat

**Review by get-itips — APPROVED** [review]

**Comment by PatAltimore** [comment]
Hi @tiburd,

This is approved and ready for merge.

Thanks!
Pat

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**